retorchfiles/scripts/waitforSUT.sh

```diff
@@ -1,19 +1,45 @@
 #!/bin/bash
+
+# Function to check the status of frontend and database
+check_status() {
+  local COUNTER=0
+  local WAIT_LIMIT=20
+  local TJOB_NAME="$1"
+  local DB_SERVICE_NAME="sqldata_$TJOB_NAME"
+  local WEB_SERVICE_URL="http://webmvc_$TJOB_NAME:80"
+  local DB_NAME="[Microsoft.eShopOnContainers.Services.CatalogDb]"
+  local TABLE_NAME="Catalog"
+  local QUERY="USE $DB_NAME; SELECT COUNT (*) FROM $TABLE_NAME;"
+
+  while true; do
+    # Check if frontend service is up
+    if curl --insecure -s "$WEB_SERVICE_URL" | grep -q "<div class=\"esh-catalog-item col-md-4\">"; then
+      # Check if database service is up
+      if docker exec "$DB_SERVICE_NAME" /opt/mssql-tools/bin/sqlcmd -S localhost -U SA -P 'Pass@word' -Q "$QUERY" | grep -q "14"; then
```

Comment:
Do not hardoce passwords nor include magic numbers that belong to the application. At the end, is this going to be checked in the tests?